webserver/src/routes/web/guards/session.rs

```diff
@@ -157,6 +157,7 @@ impl<'a, 'r> Drop for Session<'a, 'r> {
           .secure(true)
           .http_only(true)
           .same_site(SameSite::Lax)
+          .max_age(Duration::days(30))
```

Comment:
Session should not last 30 days. It should only last for the browsing session (sessions are not related to auth).